src/components/Header/Nav/VeggieBurger/index.js

```diff
@@ -0,0 +1,94 @@
+// @flow
```

Comment:
this has no siblings? should it be an `index`?

Reply:
I think I'd set it up as index with this intention of pulling the styled component`VeggieBurgerIcon` out in to a sibling, something I haven't done. I can do this.

 Sometimes it feels like overkill pulling each styled component into a separate module though, but it's a consistent approach which ensures clarity within each module.